### ISSUE TITLE
Question about MinimumLevel.Default priority

### ISSUE THREAD

**Issue by zerda** [serilog/serilog-settings-configuration#189]
Say like following code

`appsettings.json`
```json
{
  "Serilog": {
    "Using": [ "Serilog.Sinks.Console" ],
    "MinimumLevel": "Warning",
    "WriteTo": [
      { "Name": "Console" }
    ]
  }
}
```

`appsettings.Development.json`
```json
{
  "Serilog": {
    "MinimumLevel": {
      "Default": "Information"
    }
  }
}
```

And a console program `Program.cs`.
```csharp
class Program
{
  static void Main(string[] args)
  {
    var configuration = new ConfigurationBuilder()
        .AddJsonFile("appsettings.json")
        .AddJsonFile("appsettings.Development.json")
        .Build();
    var logger = new LoggerConfiguration()
        .ReadFrom.Configuration(configuration)
        .CreateLogger();

    logger.Information("It should be printed in console, but it's not.");

    Console.WriteLine("Hello World!");
  }
}
```

What I expected is `MinimumLevel` is overrided with `Information`, but actually it's `Warning`.

Is this an expected behavior?

**Comment by nblumhardt** [comment]
Hi! Yes, this is (somewhat) expected.

The `.Development` settings "override" the other settings in the sense that if two settings have the same key then `.Development` will win, but since the minimum level here is being specified under two different keys, _Microsoft.Extensions.Configuration_ will just present both of them and Serilog can't see where the two different values originated. It's just a consequence of the way that the code is written currently that the value directly under `MinimumLevel` will take precedence, but it seems as good as any option.

In short, if you want to override settings from one configuration file with values from another, for Serilog or anything else, they need to have the same keys - otherwise you'll get a mix and consumer-specific behavior.

HTH!
Nick
